specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -7873,6 +7909,62 @@
         }
       }
     },
+    "LocationGetResult": {
+      "description": "Cosmos DB location get result",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the resource group to which the resource belongs."
+        },
+        "properties": {
+          "type": "object",
+          "description": "Cosmos DB location metadata",
+          "$ref": "#/definitions/LocationProperties"
+        }
+      }
+    },
+    "LocationProperties": {
+      "description": "Cosmos DB location metadata",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "The current status of location in Azure.",
+          "type": "string"
```

Comment:
is this an enum? if so it should be modeled as such, using the num and x-ms-enum annotations